### DISCUSSION TITLE
Help for luigi user

### DISCUSSION THREAD

**Discussion by dycw** [PrefectHQ/prefect#3082] [Q&A]
Hi, I am a long time [luigi](https://github.com/spotify/luigi) user. Therefore I was pleased when I say in the [Output Caching based on a file target](https://docs.prefect.io/core/concepts/persistence.html#output-caching-based-on-a-file-target) section that luigi was mentioned.

The `LocalResult` object is not making sense to me.

```
from prefect.engine.results import LocalResult
from prefect import task, Task


# create a task via the task decorator
@task(target="func_task_target.txt", checkpoint=True, result=LocalResult(dir="~/.prefect"))
def func_task():
    return 99
```

I thought, like luigi, I would be expected to write something to `~/.prefect/func_task_target.txt` at some point. And supposedly, a downstream task would be able to read from the cached value of `func_task` automatically.

Some help please.

**Comment by joshmeek** [comment]
Hi @dycw are you running this flow locally or with a supported backend API (server or cloud)? If running local only you need to enable checkpointing for your flows by either setting this in your `.prefect/config.toml`:

```
[flows]
checkpointing = true
```

or setting the environment variable 

```bash
export PREFECT__FLOWS__CHECKPOINTING=True
```

**Reply by dycw** [reply]
Hi @joshmeek, sorry I'd missed this! I am looking to run locally. Looks like I missed this env var in the docs, let me give it a run later and report back.
